data/js/views.js

```diff
@@ -70,24 +70,26 @@ function updateView(tickerId, price, exchangeName, currency, baseCurrency, curre
     return; // Ticker was removed
   }
   updateStyle(tickerId, color, fontSize, background);
-  tickerView.text(formatTickerText(price, currency, currencyPosition));
+  tickerView.text(formatTickerText(price, currency, currencyPosition, currencyName, noRounding, baseCurrency));
   var label = exchangeName + " " + currency + "/" + baseCurrency;
   tickerView.attr("tooltiptext", label);
   tickerView.attr("title", label);
 }
 
-function formatTickerText(price, currency, currencyPosition) { // Allow more decimals for low price values
+function formatTickerText(price, currency, currencyPosition, currencyName, noRounding, baseCurrency) { // Allow more decimals for low price values
   if (price == parseFloat(price)) {
-    var roundedPrice = calculateRoundedPrice(price);
-    var tickerText = roundedPrice;
+  	if (!noRounding) { var tickerText = calculateRoundedPrice(price); }
+  	else { var tickerText = price; }
```

Comment:
The houndci-bot is right.
You can simplify these two lines of code in a ternary operator like this:
`var tickerText = noRounding ? price : calculateRoundedPrice(price);`